Makefile

```diff
@@ -84,6 +90,76 @@ else
 	@echo Update complete
 endif
 
+## zkgroup
+build-zkgroup:
+	@echo "Found cargo with version $(CARGO_VERSION)"
+	@echo "Found go with version $(GO_VERSION)"
+	@echo "Found go with version $(GIT_VERSION)"
+ifeq ($(UNAME_S), Linux)
+ifeq ($(UNAME_HARDWARE_PLATTFORM), x86_64)
+	@echo "get zkgroup $(PLATFORM)"
+	go get -d github.com/nanu-c/zkgroup
+	&& git submodule update \
+	&& cd lib/zkgroup \
+	&& cargo build --release --verbose
+	mv libzkgroup.so libzkgroup_linux_amd64.so
+else ifeq ($(UNAME_HARDWARE_PLATTFORM), aarch64)
+	@echo "get zkgroup $(PLATFORM)"
+	go get -d github.com/nanu-c/zkgroup
```

Comment:
What is the purpose of stacking all commands with `&&` like this? If it is to only execute the next part if the previous is successful, why isn't the final `mv` also conditionally executed?

Another option which to me would increase the readability and reduce complexity is to move each command to its own line and remove the usage of `&&`.